package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shikshalokam",
-  "version": "7.0.0",
+  "version": "8.0.0",
```

Comment:
Add release note for 8.0.0 @praveenKDass

Reply:
Added